test/image/mocks/layout-colorway.json

```diff
@@ -0,0 +1,32 @@
+{
+  "data": [
+    {"y": [8, 7, 7, 6, 5, 4, 4, 3, 2, 2, 3]},
+    {"y": [7, 7, 6, 5, 4, 4, 3, 2, 2, 3, 8]},
+    {"y": [7, 6, 5, 4, 4, 3, 2, 2, 3, 8, 7]},
+    {"y": [6, 5, 4, 4, 3, 2, 2, 3, 8, 7, 7]},
+    {"y": [5, 4, 4, 3, 2, 2, 3, 8, 7, 7, 6]},
+    {
+      "labels": ["a","b","c","c","c","a","d","e","f","f","g","h"],
+      "type": "pie",
+      "domain": {"x": [0, 0.4]},
+      "xaxis": "x2",
+      "yaxis": "y2"
+    }
+  ],
+  "layout": {
+    "title": "Custom Trace Color Defaults",
+    "colorway": [
+      "#DE5845",
+      "#E83898",
+      "#A83DD1",
+      "#5A38E8",
+      "#3C71DE"
+    ],
+    "xaxis": {
+      "domain": [0.4, 1]
+    },
+    "yaxis": {
+      "anchor": "y2"
```

Comment:
`anchor` for y-axes expects x-axis ids, so this thing here gets ignored. :hocho: